### PR TITLE
Update pull_request_template.md

### PR DESCRIPTION
**Pull request by JolandaVerhoef**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -3,9 +3,14 @@ name: Pull request
 about: Create a pull request
 label: 'triage me'
 ---
-Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
+Thank you for opening a Pull Request!
+Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
 - [ ] Make sure to open a GitHub issue as a bug/feature request before writing your code! That way we can discuss the change, evaluate designs, and agree on the general idea
 - [ ] Ensure the tests and linter pass
```
Comment (alexvanyo) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:9]: Could we add the Gradle command here?

```suggestion
- [ ] Ensure the tests and linter pass (`./gradlew spotlessApply` to automatically apply formatting)
```

**Review by alexvanyo — APPROVED** [review]
LGTM, just a comment for the autoformatting command.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md (hunk 1 above)
Comment (ai-erickzhao) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Auto for before it which tab for touch to get first Word.

**Review by ai-erickzhao — COMMENTED** [review]
After adding this can see in which each one

https://github.com/android/nowinandroid/pull/65#issue-1247881799

After Understanding between both is it possible to do better as before? 
Need more Guidance

**Pull request merged**